src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1191,23 +1191,23 @@
     <comment>Header for a control to how text is formatted</comment>
   </data>
   <data name="Appearance_IntenseTextStyle.Header" xml:space="preserve">
-    <value>Intense Text Format</value>
+    <value>Intense (Bold) Text Format</value>
     <comment>Header for a control to select how "intense" text is formatted (bold, bright, both or none)</comment>
   </data>
   <data name="Appearance_IntenseTextStyleNone.Content" xml:space="preserve">
     <value>None</value>
     <comment>An option to choose from for the "intense text format" setting. When selected, "intense" text will not be rendered differently</comment>
   </data>
   <data name="Appearance_IntenseTextStyleBold.Content" xml:space="preserve">
-    <value>Bold</value>
+    <value>Show bold with a heavier font weight</value>

```

Comment:
I personally find this confusing: This option doesn't show bold text in a heavier font weight, it enables having any bold for intense text in the first place.
I also suspect this might lead to awkward translations. In Simplified Chinese this leads to "Show bold, font weight is heavier". German is alright, but has the same logical problem as the English variant.

In some way I liked the previous variant more. As far as I understood it none of the other terminals are that verbose about it, right?

Reply:
What are your thoughts on something like "Show intense/bold with a heavier font weight"? 

Also paging @cinnamon-msft for wording ideas

Reply:
Personally I'm fine with it being succinct.
"bold" is the technical term for "heavier font weight", so adding "intense" doesn't make it better in my opinion: It's still akin to "Show bold as bold".
I don't have a good idea for a better description than what we have now.

Reply:
What about something like:

Intense Text Style
* Bold font
* Bright colors
* Bold font with bright colors
* None

This follows what we have in the JSON plus it should indicate to the user what "intense" means when they see the options underneath.

Also do we need the "Text Formatting" header? I'm assuming this is under the Text category on the Profile Appearance page anyway?

Reply:
we may just want to not offer `none`

this is also the first of possibly many _format-specific_ options like "do italics work? do you want underlines to work? overlines?" I kinda.. don't hate that we have a section for it.

@cinnamon-msft FYI James did a quick comp against some of our peer terminals and found that they had slightly more verbose wording: https://github.com/microsoft/terminal/issues/10955#issue-971702089

Reply:
> this is also the first of possibly many format-specific options like "do italics work? 

crazy that you say that - it's exactly why I gave it a whole section. I could have sworn there was a feature request for "disable italics" but I couldn't find it.